README.md

```diff
@@ -1,196 +1,146 @@
+    version: [">=0.8.0", "<0.9.0"]
 
-> **Note**: All new Google Ads Fivetran connectors will be setup with the Google Ads API. If you are unable to infer the API type based on the above steps, it is most likely that you are using the Google Ads API.
-### Google Ads API Configuration
-If your connector is setup using the Google Ads API then you will need to configure your `dbt_project.yml` with the below variable:
+```
+## Step 3: Define database and schema variables
+By default, this package runs using your destination and the `google_ads` schema. If this is not where your Google Ads data is (for example, if your Google Ads schema is named `google_ads_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-  api_source: google_ads  ## google_ads by default, but may be changed to 'adwords' if using a previous version of the connector.
+    google_ads_database: your_destination_name
+    google_ads_schema: your_schema_name 
 ```
 
-### Adwords API Configuration
-If your connector is setup using the Adwords API then you will need to pull the following custom reports through Fivetran:
-
-* Destination Table Name: `final_url_performance`
-* Report Type: `FINAL_URL_REPORT`
-* Fields:
-  * AccountDescriptiveName
-  * AdGroupId
-  * AdGroupName
-  * AdGroupStatus
-  * CampaignId
-  * CampaignName
-  * CampaignStatus
-  * Clicks
-  * Cost
-  * Date
-  * EffectiveFinalUrl
-  * ExternalCustomerId
-  * Impressions
-
-* Destination Table Name: `criteria_performance`
-* Report Type: `CRITERIA_PERFORMANCE_REPORT`
-* Fields:
-  * AccountDescriptiveName
-  * AdGroupId
-  * AdGroupName
-  * AdGroupStatus
-  * CampaignId
-  * CampaignName
-  * CampaignStatus
-  * Clicks
-  * Cost
-  * Criteria
-  * CriteriaDestinationUrl
-  * CriteriaType
-  * Date
-  * ExternalCustomerId
-  * Id
-  * Impressions
-
-* Destination Table Name: `click_performance`
-* Report Type: `CLICK_PERFORMANCE_REPORT`
-* Fields:
-  * AccountDescriptiveName
-  * AdGroupId
-  * AdGroupName
-  * AdGroupStatus
-  * CampaignId
-  * CampaignName
-  * CampaignStatus
-  * Clicks
-  * CriteriaId
-  * Date
-  * ExternalCustomerId
-  * GclId
-
-The package assumes that the corresponding destination tables are named `final_url_performance`, `criteria_performance`, and `click_performance` respectively. If these tables have different names in your destination, enter the correct table names in the `google_ads__final_url_performance`, `google_ads__click_performance`, and `google_ads__criteria_performance` variables so that the package can find them:
+## (Optional) Step 4: Additional configurations
+
+### Adding passthrough metrics
+By default, this package will select `clicks`, `impressions`, and `cost` from the source reporting tables to store into the staging models. If you would like to pass through additional metrics to the staging models, add the following configuration to your `dbt_project.yml` file:
```

Comment:
Can we also add in a little blurb here about doing some due diligence before including/interpreting additional metrics? I want to make sure we have it documented that the metrics that we've included have been vetted for aggregation, however we can't ensure that all additional passthrough metrics will make sense with aggregation

Reply:
Great call out. I just added a not to this section.